modules/openapi-generator/src/test/resources/3_0/issue_20079_api_default_test.go

```diff
@@ -0,0 +1,70 @@
+package openapi
```

Comment:
thanks for the PR and adding the tests.

can you please PM me to discuss the integration tests when you've time?

Slack: https://join.slack.com/t/openapi-generator/shared_invite/zt-2wmkn4s8g-n19PJ99Y6Vei74WMUIehQA